h/session.py

```diff
@@ -66,7 +66,7 @@ def _current_groups(request, authority):
     authority_groups = (request.find_service(name='authority_group')
                         .public_groups(authority=authority))
 
-    groups = authority_groups + _user_groups(user)
+    groups = set(authority_groups + _user_groups(user))
```

Comment:
Given the discussion that’s already happened on this pull request, this might be a moot point by now, but does this version still preserve the ordering? As I understand it, the `set` type makes no promises about the order of its elements, so might this result in the groups being displayed out of order in the client?